galata/README.md

```diff
@@ -79,7 +79,7 @@ c.LabApp.expose_app_in_browser = True
 Then start the server with:
 
 ```bash
-jupyter lab --config jupyter_server_test_config.py
+jupyter lab --LabApp.config jupyter_server_test_config.py
```

Comment:
As mentioned by @fcollonval in https://github.com/jupyterlab/jupyterlab/pull/11391#issuecomment-957384056, do we need the explicit `LabApp`?